tools/building.py

```diff
@@ -1338,3 +1338,14 @@ def js_legalization_pass_flags():
     # assumes they are imports.
     flags += ['--pass-arg=legalize-js-interface-exported-helpers']
   return flags
+
+
+# Returns a list of flags to pass to emcc that make the output run properly in
+# the given node version.
+def get_emcc_node_flags(node_version):
```

Comment:
This function name makes me thing its generated node flags.  Not sure I can come up with a better name though.